SimpleNtuplizer/plugins/SimpleNtuplizer.cc

```diff
@@ -511,11 +541,12 @@ void SimpleNtuplizer::analyze( const edm::Event& iEvent, const edm::EventSetup&
       break;
     }
   }
-      
+  
   // Determine number of primary vertices
-  if (vertices->empty()) nPV_ = 0;
-  else nPV_ = vertices->size();
-
+  if(!doPFTree){
```

Comment:
I would prefer that you added another bool that tells whether or not to record nvtx. We will fix this in the skim in the future.